docs/user-guide/generate-certificates.md

```diff
@@ -214,18 +211,13 @@ After you configure the `zowe.yaml`, use the following procedure to generate a J
 
 2. Run the following command in the directory with this `zowe.yaml` in terminal to generate the certificate and update the configuration values in `zowe.yaml`.
 
-    `zwe init certificate -c ./zowe.yaml --update-config`
-
-    Running this command generates the following artifacts:
-    * A customized JCL member name is created in the `CUST.JCLLIB` data set. 
-    * The PDS name is defined in the `zowe.setup.dataset.jcllib` property. 
-    
-    In the following example output, the PDS member `USER.ZWEV2.CUST.JCLLIB(ZW101431)` is created that contains the security manager commands, and then submitted as a job ID: `ZWEKRING(JOB03054)`.
+   `zwe init certificate -c ./zowe.yaml --update-config`
 
-    The following command output shows the generation of a JCERACFKS certificate using the default values. 
+    When the command is run, a customized JCL member name is created in the `CUST.JCLLIB` data set. The PDS name is defined in the `zowe.setup.dataset.jcllib` property. In the following example output, the PDS meember `USER.ZWEV2.CUST.JCLLIB(ZW101431)` is created that contains the security manager commands, and then submitted as a job ID: `ZWEKRING(JOB03054)`.
 
-    **Note:** Some detailed output messages have been omitted.
+The following command output shows the generation of a JCERACFKS certificate using the default values. Note that some detailed output messages have been omitted.
 
+**Command output:**
 ```
 #>zwe init certificate -c ./zowe.yaml --update-config
```

Comment:
Same comment as I had previously, so probably do a replace all, text search for "-c ./zowe.yaml" and replace them with "-c <path-to-your-zowe-configuration-yaml>" in docs-site repo

Might be dangerous, but probably not. Would need a glance